Add onOpen(:) to watch Combine call (#6833)
realm/realm-cocoa: CHANGELOG.md

x.y.z Release notes (yyyy-MM-dd)
=============================================================
### Enhancements

> * Add ability to listen for when a Watch Change Stream is opened when using Combine. Use `onOpen(event:)` directly after opening a `WatchPublisher`, this will allow a callback to be invoked once the change stream is opened.

Reviewer: ```suggestion
* Add the ability to listen for when a Watch Change Stream is opened when using Combine. Use `onOpen(event:)` directly after opening a `WatchPublisher` to register a callback to be invoked once the change stream is opened.
```

The second sentence is a comma splice.
